packages/modules/src/modules/banners/choiceCardsBanner/ChoiceCardsBanner.stories.tsx

```diff
@@ -135,6 +136,7 @@ ChoiceCardsBannerBlue.args = {
     },
     backgroundColor: blueBannerBackgroundColor,
     headingColor: blueBannerHeadingColor,
+    borderTop: true,
```

Comment:
Then to follow the existing convention you could import the exported `borderTop` variable from the Blue banner component using an alias prefixed with "blueBanner". 

You'll also need to destructure the borderTop property on line 146 in this file to remove it for the yellow banner story

Reply:
Yes did this too.